local-testnet/network_params.yaml

```
@@ -0,0 +1,17 @@
+# Full configuration reference [here](https://github.com/kurtosis-tech/ethereum-package?tab=readme-ov-file#configuration).
+participants:
+  - el_type: geth
+    el_image: ethereum/client-go:latest
+    cl_type: lighthouse
+    cl_image: sigp/lighthouse:latest
+    cl_extra_params:
+      - --target-peers=3
+    validator_count: 80
+    count: 4
+network_params:
+  deneb_fork_epoch: 0
+  seconds_per_slot: 3
+  preregistered_validator_keys_mnemonic: "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete"
+global_log_level: info
```

```suggestion
keymanager_enabled: true
global_log_level: info
```
we need to expose the VC API, enabling `keymanager` is the proper way to do so

---

Good catch! Forgot about that. Thanks!